recipes/emacs/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]; then
+  OPTS="--without-x"
+else
+  OPTS="--x-includes=$PREFIX/include --x-libraries=$PREFIX/lib"
+fi
+
+bash configure  --prefix=$PREFIX $OPTS
```

Comment:
Calling `bash` did not work on `OS X`. One alternative is to `chmod +x configure` or try `sh configure`.

The weird thing is that the source is already... Not sure what is going on.